pandas/tests/reshape/test_merge.py

```diff
@@ -724,6 +724,63 @@ def test_indicator(self):
                           how='outer', indicator=True)
         assert_frame_equal(test5, hand_coded_result)
 
+    def test_validation(self):
+        left = DataFrame({'a': ['a', 'b', 'c', 'd'],
+                          'b': ['cat', 'dog', 'weasel', 'horse']},
+                         index=range(4))
+
+        right = DataFrame({'a': ['a', 'b', 'c', 'd', 'e'],
+                           'c': ['meow', 'bark', 'um... weasel noise?',
+                                 'nay', 'chirp']},
+                          index=range(5))
+
+        merge(left, right, left_index=True, right_index=True, validate='1:1')
```

Comment:
Good point. I actually should test those come to think of it -- in theory no impact on output (it's just tests), but no reason to make that a tested behavior.